librz/bin/p/bin_qnx.c

```diff
@@ -89,7 +122,7 @@ static bool load_buffer(RzBinFile *bf, RzBinObject *obj, RzBuffer *buf, Sdb *sdb
 			}
 			ptr->name = strdup("LMF_RESOURCE");
 			ptr->paddr = offset;
-			ptr->vsize = lrec.data_nbytes - sizeof(lmf_resource);
+			ptr->vsize = lrec.data_nbytes - LMF_RESOURCE_SIZE;
```

Comment:
add a check for `lrec.data_nbytes being >= LMF_RESOURCE_SIZE` because this might come out when fuzzing.